db/migrate/20211213040746_update_account_summaries_to_version_2.rb

```diff
@@ -0,0 +1,24 @@
+class UpdateAccountSummariesToVersion2 < ActiveRecord::Migration[6.1]
+  def up
+    reapplication_follow_recommendations_v2 do
+      drop_view :account_summaries, materialized: true
+      create_view :account_summaries, version: 2, materialized: { no_data: true }
+      safety_assured { add_index :account_summaries, :account_id, unique: true }
+    end
+  end
+
+  def down
+    reapplication_follow_recommendations_v2 do
+      drop_view :account_summaries, materialized: true
+      create_view :account_summaries, version: 1, materialized: { no_data: true }
+      safety_assured { add_index :account_summaries, :account_id, unique: true }
+    end
+  end
+
+  def reapplication_follow_recommendations_v2
+    drop_view :follow_recommendations, materialized: true
+    yield
+    create_view :follow_recommendations, version: 2, materialized: { no_data: true }
+    safety_assured { add_index :follow_recommendations, :account_id, unique: true }
+  end
+end
```

Comment:
I wonder if, instead of the `reapplication_follow_recommendations_v2` thing, you can use `replace_view` instead? Since the columns are exactly the same in the same order, I think this should work.

Reply:
Actually, no this is not possible. This is only possible for non-materialized views.

Reply:
I'm looking forward to scenic's update_view reducing this verbose description, but not yet.
https://github.com/scenic-views/scenic/pull/218